src/main/java/resources/PersonResource.java

```diff
@@ -72,6 +72,13 @@ public List fetchPersonsWithSlots(@QueryParam("startDate") String startDate, @Qu
     }
 
     @GET
+    @Path("/all/withoutSlots")
+    @UnitOfWork
+    public List fetchPersonsWithoutSlots() {
```

Comment:
is this named correctly? we are not fetching persons without slots, but all persons